cwms-data-api/src/main/java/cwms/cda/data/dto/binarytimeseries/BinaryTimeSeriesRow.java

```diff
@@ -0,0 +1,193 @@
+package cwms.cda.data.dto.binarytimeseries;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.Arrays;
+import java.util.Date;
+
+@JsonDeserialize(builder = BinaryTimeSeriesRow.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class BinaryTimeSeriesRow {
+    private final Date dateTime;
+    private final Date versionDate;
+    private final Date dataEntryDate;
+    private final String binaryId;
+    private final Long attribute;
+    private final String mediaType;
+    private final String fileExtension;
+    private final byte[] binaryValue;
```

Comment:
Not entirely sure why, but my codegen returns this as a `List<byte[]>`. I don't know what we can do about that

Reply:
curious.  Is all the data in the first list item?  It doesn't spread it out or duplicate it

Reply:
It won't deserialize because the actual returned JSON only has the byte array. I've just hand-edited the codegenned file as a workaround.

Reply:
the schema comes out like:
```
        binary-value:
          type: array
          items:
            type: string
            format: byte
```
Which is what I'd expect